src/main/java/core/basesyntax/IsoscelesTrapezoidSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class IsoscelesTrapezoidSupplier {
+    private Random random = new Random();

```

Comment:
The 'random' variable is declared but not used anywhere in the class.

Reply:
Done it.